src/hooks/useComponentEventListenersPlugin.tsx

```diff
@@ -1,46 +1,10 @@
-import { Plugin, PluginKey } from "prosemirror-state";
```

Comment:
Should we drop plugin from the name of this file and hook, too?

Reply:
Yeah!